src/Riter/Core/GlobalHotkeyManager.cs

```diff
@@ -105,11 +105,11 @@ private IntPtr HwndHook(IntPtr hwnd, int msg, IntPtr wParam, IntPtr lParam, ref
 
         if (msg == wM_HOTKEY)
         {
-            var id = wParam.ToInt32();
+            var id = (HotKey)wParam.ToInt32();
 
             if (_hotkeyActions.TryGetValue(id, out var value))
```

Comment:
Abstract the hotkey processing logic by using a strategy pattern or delegating it to another class.

in new issue.

Reply:
like IMessageHandler